pyfar/dsp/dsp.py

```diff
@@ -1929,6 +1930,19 @@ def average(signal, mode='linear', caxis=None, weights=None, keepdims=False):
         If this is ``True``, the axes which are reduced during the averaging
         are kept as a dimension with size one. Otherwise, singular dimensions
         will be squeezed after averaging. The default is ``False``.
+    nan_policy: string, optional
+        Define how to handle NaNs in input signal.
+
+        ``'propagate'``
+           If the input signal includes NaNs, the resulting averaged output
+           signal values are NaN.
+        ``'omit'``
+           NaNs will be omitted while averaging.
```

Comment:
For me, it would not be directly clear, that this refers both to the numerator and denumerator in the average operation. (so average of `[1,1,np.nan]` is 2 and not 2/3). But I might be the only one...